pep/app/styles/_pep-documents.scss

```diff
@@ -24,3 +24,17 @@
         background-color: transparent;
     }
 }
+
+// used in places like "expert pick of the day"
+// hides some known elements of the abstract HTML
+// so only the main body content is displayed
+.pep-abstract-preview {
+    p.title,
+    p.title_author,
+    p.figure,
+    p.pagenumber,
+    p:empty,
+    pb {
+        display: none;
+    }
+}
```

Comment:
you removed this class from the `home` template, but we are adding back the styles? I thought we decided to get rid of this.

Reply:
We need it for just the abstract preview widget